JS-basics/js-basics_dark-mode/js/index.js

```diff
@@ -1,3 +1,21 @@
 console.clear();
 
 const bodyElement = document.querySelector('[data-js="body"]');
+const dark = document.querySelector('[data-js="dark-mode-button"]');
+const light = document.querySelector('[data-js="light-mode-button"]');
+const toggle = document.querySelector('[data-js="toggle-button"]');
+
+dark.addEventListener("click", () => {
+  bodyElement.classList.add("button--light");
```

Comment:
line 9 actually adds light mode when you click on the button? Why? we only want to add dark mode. you dont need that code